tutorials/introductory/customizing.py

```diff
@@ -214,12 +227,13 @@ def plotting_function():
 #   >>> matplotlib.matplotlib_fname()
 #   '/home/foo/.config/matplotlib/matplotlibrc'
 #
-# See below for a sample :ref:`matplotlibrc file<matplotlibrc-sample>`.
+# See below for a sample :ref:`matplotlibrc file<matplotlibrc-sample>`
+# and see `matplotlib.rcParams` for a full list of configurable rcParams.
 #
 # .. _matplotlibrc-sample:
 #
 # A sample matplotlibrc file
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# --------------------------
```

Comment:
While you are here change this section heading to point out that is the default defaults?

Reply:
Renamed section to `The default matplotlibrc file`